src/plugins/CMakeLists.txt

```diff
@@ -67,6 +67,19 @@ ELSE ()
           COMMAND cmake -E copy ./example/config/taosadapter.toml ${CMAKE_BINARY_DIR}/test/cfg/
           COMMAND cmake -E copy ./taosadapter.service ${CMAKE_BINARY_DIR}/test/cfg/
           )
+      ExternalProject_Add(taosadapter-debug
+          PREFIX        "taosadapter-debug"
```

Comment:
LGTM. But seems no difference with the next change block (ignore the os cmake condition).